types.d.ts

```diff
@@ -5,6 +5,7 @@
  */
 export type License = SPDXLicense | CustomLicense;
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Why this has been disabled?

Reply:
I strongly recommend you please with XO on your local so that you can know why this was added.

We're using some `UPPER_CASE` types/variables, and it's exported to our users. We can change its case right now.

Reply:
I've tested and I can't see errors running xo.

Reply:
![CleanShot 2024-03-19 at 20 15 16@2x](https://github.com/simple-icons/simple-icons/assets/8186898/bed6d02a-c0a3-487b-bcf5-4e96b3fb3d6e)

It's recommended case is `SpdxLicense`.

Reply:
Ah OK, that would be a breaking. Fine enough :+1: Sorry it was not being executed locally.